src/odemis/acq/stream/test/stream_sparc2_streakcam_test.py

```
@@ -825,6 +835,101 @@ def test_streak_acq_integrated_images_leech(self):
+        streaks.detPcIntegrationCounts.value = 100
+        streaks.detPhotonCounting.value = True
+
+        expected_exp_time = streaks.detPcExposureTime.value * streaks.detPcIntegrationCounts.value
+
+        # Disable the protections
+        streaks.detMCPGain.value = 5
+        streaks.detShutter.value = False
+
+        streaks.repetition.value = (7, 3)
+        num_ts = numpy.prod(streaks.repetition.value)  # number of expected temporal spectrum images
+        exp_pos, exp_pxs, exp_res = roi_to_phys(streaks)
```

Ruff (0.16.1)

[warning] Unpacked variable `exp_pos` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)